unit-tests/shared/src/test/scala/javalib/lang/RuntimeTest.scala

```diff
@@ -7,30 +7,42 @@ import java.io.File
 
 import org.junit.Test
 import org.junit.Assert._
+import org.junit.Assume._
+import org.scalanative.testsuite.utils.Platform._
 
 class RuntimeTest {
   import ProcessUtils._
+  private val EOL = System.lineSeparator()
+  private def lsCommand =
+    if (isWindows)
+      Array("cmd", "/c", "dir", "/b")
+    else
+      Array("ls")
+
   @Test def execCommand(): Unit = {
-    val proc = Runtime.getRuntime.exec(Array("ls", resourceDir))
+    val proc = Runtime.getRuntime.exec(lsCommand :+ resourceDir)
     val out = readInputStream(proc.getInputStream)
     assertTrue(proc.waitFor(5, TimeUnit.SECONDS))
-    assertTrue(
-      out.split("\n").toSet == Set("echo.sh", "err.sh", "hello.sh", "ls")
-    )
+    assertEquals(Scripts.values.map(_.filename), out.split(EOL).toSet)
   }
   @Test def execEnvp(): Unit = {
+    assumeFalse(
+      "Not possible in Windows, would use dir keyword anyway",
+      isWindows
+    )
+    assumeFalse("Not complient with JVM", executingInJVM)
```

Comment:
```suggestion
    assumeFalse("Not compliant with JVM", executingInJVM)
```
Also in some other places.